pkg/basicstation/cups/server.go

```diff
@@ -106,18 +106,28 @@ func (s *Server) getAuth(ctx context.Context, eui types.EUI64, auth string) grpc
 	return s.component.WithClusterAuth()
 }
 
-func (s *Server) getRegistry(ctx context.Context, ids *ttnpb.GatewayIdentifiers) ttnpb.GatewayRegistryClient {
+var errESUnavailable = errors.DefineUnavailable("entity_registry_unavailable", "Entity Registry unavailable for gateway_id `{gateway_id}`")
```

Comment:
- `errERPeerUnavailable` or `errEntityRegistryPeerUnavailable`
- no code names in human friendly error messages (i.e. no `gateway_id` but `gateway`)
- always UIDs in error messages, so `{gateway_uid}` or `{uid}` here